lib/liftoff/cli.rb

```diff
@@ -7,7 +7,7 @@ def initialize(argv)
 
     def run
       parse_command_line_options
-      LaunchPad.new.liftoff @options
+      LaunchPad.new.liftoff @options, @argv.first
```

Comment:
If you set this at the very bottom of the option parsing, you'll be able to remove a lot of the logic around adding this to the configuration. Should be as simple as `@options[:path] = @argv.first`